coalib/processes/Processing.py

```diff
@@ -344,7 +347,35 @@ def instantiate_processes(section,
         log_printer,
         ignored_file_paths=glob_list(section.get('ignore', "")),
         limit_file_paths=glob_list(section.get('limit_files', "")))
+
+    # This stores all matched files irrespective of whether coala is run
+    # only on changed files or not. Global bears require all the files
+    complete_filename_list = filename_list
+
+    # Start tracking all the files
+    if cache:
```

Comment:
if cache is None `cache.get_uncached_files` below won't work, right?

I wouldn't allow cache to be none, makes everything really complicated.